src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -408,11 +409,14 @@ void uploadTseSupportingDocument(CaseDetails caseDetails, UploadedDocumentType c
             docList = new ArrayList<>();
         }
 
+        String contactAppType = APP_TYPE_MAP.get(contactApplicationType);
+        String applicationType = VARY_OR_REVOKE_ORDER_APP_TYPE.equals(
+            contactAppType) ? contactAppType.replace("/", " or ") : contactAppType;
```

Comment:
Just from a readability perspective, If it's just the one application that needs some text changed, would it be better to define it as a constant instead of having the replace method?

Reply:
Rather than contactAppType.replace("/", " or "), a constant where its "Vary or revoke an order"